rpc/trace_test.go

```diff
@@ -382,15 +398,19 @@ func TestTraceTransactionV0_6(t *testing.T) {
 		require.NoError(t, json.Unmarshal(vmTraceJSON, vmTrace))
 		mockVM.EXPECT().Execute([]core.Transaction{tx}, []core.Class{declaredClass.Class}, []*felt.Felt{},
 			&vm.BlockInfo{Header: header}, gomock.Any(), &utils.Mainnet, false, false, false, false).
-			Return(nil, nil, []vm.TransactionTrace{*vmTrace}, nil)
+			Return(nil, nil, []vm.TransactionTrace{*vmTrace}, uint64(123), nil)
 
-		trace, err := handler.TraceTransactionV0_6(context.Background(), *hash)
+		trace, httpHeader, err := handler.TraceTransactionV0_6(context.Background(), *hash)
 		require.Nil(t, err)
+		require.NotNil(t, httpHeader)
+		require.Equal(t, httpHeader.Get(rpc.ExecutionStepsHeader), "123")
 		assert.Equal(t, vmTrace, trace)
 	})
 }
 
-func TestTraceBlockTransactionsV0_6(t *testing.T) {
+func TestTraceBlockTransactions(t *testing.T) {
```

Comment:
why changing name of this test? and calling function below?

Reply:
came as a result of a merge from main.